openqa-qam-sle

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/env bash
+
+set -e
+
+usage() {
+    cat << EOF
+Usage: $(basename "$0") -a SUSE:Maintenance:II:RR [-d days]
+
+Search openQA jobs related to a given update inside job groups:
```

Comment:
Do you mean "Search *for* openQA jobs"? Because in https://github.com/os-autoinst/scripts/pull/326#issuecomment-2222987460 you stated the opposite that you would start with a job-id but here you search by incident+RR which I explained is better done with the /job_settings/jobs api route

Reply:
There are two points here:
1. I prefer not to have any dependencies with other tools, so the script can be run even if openqa-cli is not installed
2. is there an equivalent API URL that can be called with a simple curl command?

Said that, I am going to modify the description so that could explain better the goal of this script, that is basically to fill the section "regression tests" in the log of QA test report.

Reply:
> There are two points here:
> 
>     1. I prefer not to have any dependencies with other tools, so the script can be run even if openqa-cli is not installed

Sure, that's fine and certainly possible with the way *u implemented using only curl and openQA API routes
> 
>     2. is there an equivalent API URL that can be called with a simple curl command?

Well I think the API route /job_settings/jobs which can be called just with curl should suffice. If there is something missing I would be happy to learn about that.

> Said that, I am going to modify the description so that could explain better the goal of this script, that is basically to fill the section "regression tests" in the log of QA test report.

Good idea!

Reply:
Changed description